react-forms/my-app/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,33 @@
+export function RegistrationFormUncontrolled() {
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>): void {
+    event.preventDefault();
+
+    const $form = event.currentTarget;
```

Comment:
In React we don't use the leading `$` to indicate DOM elements. In fact, in React you rarely interact directly with DOM elements.